warggroundstation

```diff
@@ -0,0 +1 @@
+Subproject commit 684ad1dfd95a8025f05ba8f5afdead6c17c57f4d
```

Comment:
As mentioned on other PR, this has nothing to do with netbeans and is a
valid git file that should be committed (when using git submodules).
On Mar 4, 2016 2:30 PM, "Serge Babayan" <notifications@github.com> wrote:

> In warggroundstation
> <https://github.com/UWARG/WARG-Ground-Station/pull/112#discussion_r55099056>
> :
>
> > @@ -0,0 +1 @@
> > +Subproject commit 684ad1dfd95a8025f05ba8f5afdead6c17c57f4d
>
> Get rid of this thing. Netbeans sucks.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UWARG/WARG-Ground-Station/pull/112/files#r55099056>.
>